kaldb/src/main/proto/trace.proto

```diff
@@ -18,9 +19,10 @@ import "google/protobuf/empty.proto";
 enum ValueType {
   STRING  = 0;
   BOOL    = 1;
+  INT32   = 5;
   INT64   = 2;
   FLOAT64 = 3;
-  BINARY  = 4;
```

Comment:
Is there a reason to drop the Binary type? I think it has a lot of value to keep raw data unmodified in Lucene. We should keep it.